python/.bumpversion.cfg

```diff
@@ -0,0 +1,18 @@
+[bumpversion]
+current_version = 0.0.5
```

Comment:
The idea is changing version here, it will automatically upgrade `pylance.__version__`?

Reply:
yeah, when you call bump version, it should automatically change `lance/version.py`, which is used both in setup.py and lance.__version__